### PR TITLE
[DT-887] fix: reverse resolution records saving

### PR DESCRIPTION
**Pull request by DancingAxolotl**
## Background
Resolution service returns incorrect reverse resolutions when the reverse address is set multiple times. This happens because the old reverse resolution is not removed from db when we add a new one.

### Linear issue
https://linear.app/unstoppable-domains/issue/DT-887/resolution-service-returns-incorrect-reverse-resolutions

## Changes
 - Update `processSetReverse` in `EthUpdater`
 - Update tests

### CODE REVIEW

**Review by sammyluo — APPROVED** [review]
🙌

File: src/database/migrations/1664366869085-CleanReverseResolutions.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CleanReverseResolutions1664366869085
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
```
Comment (enaqx) [inline src/database/migrations/1664366869085-CleanReverseResolutions.ts:6]: If this migration does not change db schema and is non-reversible mayve it is better to run directly on DB without migration?

Comment (DancingAxolotl) [reply src/database/migrations/1664366869085-CleanReverseResolutions.ts:6]: Makes sense. I'll add it to `scripts` instead to preserve the query. Thanks!

**Review by enaqx — APPROVED** [review]
LGTM! 👍

**Pull request merged**